site/src/pages/UserSettingsPage/AppearancePage/AppearanceForm.tsx

```diff
@@ -128,6 +149,35 @@ export const AppearanceForm: FC<AppearanceFormProps> = ({
 					</RadioGroup>
 				</FormControl>
 			</Section>
+			<div css={{ marginBottom: 48 }}></div>
+			<Section
+				title={
+					<Stack direction="row" alignItems="center">
+						<span>Terminal Font Size</span>
+						{isUpdating && <CircularProgress size={16} />}
+					</Stack>
+				}
+				layout="fluid"
+			>
+				<TextField
```

Comment:
@BrunoQuaresma I'll need your design help here.

<img width="1089" alt="Screenshot 2025-04-09 at 16 24 31" src="https://github.com/user-attachments/assets/918b3435-9516-4edc-9004-d9c748e3eb19" />

I have troubles with validation. Currently, the page submits the form "on Blur", but unfortunately user can manually type an incorrect value (for example: 33), and the page will display an ugly validation error. Would you mind giving some advice on how we can make this form less awkward?

Ideally, the input component should accept values between 8 and 32, and reject/revert other wrong inputs.

PS. Feel free to modify the page if needed!